cooked-validators/src/Cooked/Tx/Constraints/Type.hs

```diff
@@ -665,14 +907,12 @@ data TxSkel where
       txSkelMints :: TxSkelMints,
       txSkelValidityRange :: Pl.POSIXTimeRange,
       txSkelRequiredSigners :: Set Pl.PubKeyHash,
-      txSkelIns :: Map SpendableOut TxSkelIn,
+      txSkelIns :: Map Pl.TxOutRef TxSkelRedeemer,
```

Comment:
As far as I can see, there's no mechanism to prevent associating for instance a pk utxo with the "NoRedeemerForScript" redeemer type (or any inconsistent association). Is that handled somewhere? Could it yield issues at some point?